core/src/main/java/hudson/FilePath.java

```diff
@@ -1323,8 +1323,10 @@ public String getName() {
     /**
      * Short for {@code getParent().child(rel)}. Useful for getting other files in the same directory. 
```

Comment:
```suggestion
     * Gets a sibling file in the same directory.
     * @param rel name of the file
     * @return Sibling file with a specified file name.
     *         {@code null} if no sibling file can be created, e.g. if the current file points to the filesystem root.
```

Copied from https://github.com/jenkinsci/jenkins/pull/4526